lib/android.js

```diff
@@ -288,7 +288,12 @@ function _getApi () {
       '_ZN3art31InternalDebuggerControlCallback13StartDebuggerEv',
       '_ZN3art3Dbg15gDebuggerActiveE',
       '_ZN3art15instrumentation15Instrumentation20DeoptimizeEverythingEPKc',
-      '_ZN3art15instrumentation15Instrumentation20DeoptimizeEverythingEv'
+      '_ZN3art15instrumentation15Instrumentation20DeoptimizeEverythingEv',
+      '_ZN3art3Dbg9StartJdwpEv',
+      '_ZN3art3Dbg8GoActiveEv',
+      '_ZN3art3Dbg21RequestDeoptimizationERKNS_21DeoptimizationRequestE',
+      '_ZN3art3Dbg20ManageDeoptimizationEv',
+      '_ZN3art3Dbg9gRegistryE'
```

Comment:
Are these all gone without any replacement?

Reply:
To the best of my understanding~yeah!
I made a diff comparison between:
- https://android.googlesource.com/platform/art/+/android-10.0.0_r27/runtime/debugger.cc
- https://android.googlesource.com/platform/art/+/master/runtime/debugger.cc

and it seems there are no replacements, or maybe you can correct me on this.

Reply:
To the best of my understanding~yeah!
I made a diff comparison between:

- https://android.googlesource.com/platform/art/+/android-10.0.0_r27/runtime/debugger.cc
- https://android.googlesource.com/platform/art/+/master/runtime/debugger.cc

and it seems there are no replacements, or maybe you can correct me on this.

Reply:
Ok! Thanks for checking. We can always deal with these later.